client/src/index.js

```diff
@@ -58,6 +72,8 @@ const Routes = () => {
             path="/sign-up-interpreter/"
             component={InterpreterSignUpForm}
           />
+
+          <Route path="/add-document" component={AddDocumentForm} />
```

Comment:
This should be also added conditionnally, otherwise a user could enter the url in the browser manually and access the page. To avoid doing again sessionStorage.getItem("token"), you could create a new function isLoggedIn() that you can use in different places.